chaste_codegen/__init__.py

```diff
@@ -35,5 +36,21 @@
 del(logging)
 
 # Set cellmlmanip exp function
-setattr(sympy, '_exp', sympy.Function('_exp'))
+setattr(sympy, '_exp', _exp)
 transpiler.SIMPLE_MATHML_TO_SYMPY_NAMES['exp'] = '_exp'
+
+setattr(sympy, '_abs', _abs)
+transpiler.SIMPLE_MATHML_TO_SYMPY_NAMES['abs'] = '_abs'
+transpiler.SIMPLE_MATHML_TO_SYMPY_NAMES['Abs'] = '_abs'
```

Comment:
Why on earth is this needed?

Reply:
the 1st line makes sp._abs available
the 2nd line the transpiler to use sympy._abs.  when it encounters abs in the cellml